WalletWasabi/Blockchain/TransactionBuilding/BuildTransactionResult.cs

```diff
@@ -26,7 +26,7 @@ public BuildTransactionResult(SmartTransaction transaction, PSBT psbt, bool sign
 	public Money Fee { get; }
 	public decimal FeePercentOfSent { get; }
 	public bool SpendsUnconfirmed => Transaction.WalletInputs.Any(c => !c.Confirmed);
-
+	public bool SpendsCoinjoining => Transaction.WalletInputs.Any(c => c.CoinJoinInProgress);
```

Comment:
IMO the name `SpendsCoinjoiningCoins` would more clear. Can you rename it?

Reply:
@adamPetho Give me a hand, isn't the plan to remove `CoinJoinInProgress` flag from SmartCoin?

Reply:
Well, in the long run yes. I have a PR about this https://github.com/zkSNACKs/WalletWasabi/pull/10923, but it's only refactoring and it's unrelated to my tasks right now, so it's parking.

I was aware of this change, but I didn't raise any concerns, because we use the `CoinJoinInProgress` a lot in the codebase and I will eliminate them in one go in the (hopefully soonish) future.

Reply:
Now that I'm thinking about this, I'm even fine if you don't do my suggested change.
I can do it later on in my own PR. Feel free to leave it as it is.

Reply:
@ichthus1604 ok. I also noticed that it's "Unconfirmed**Funds**Warning" etc, while in the GUI it was updated to _coins_ . I can update it in follow up PR